auth/src/main/java/com/firebase/ui/auth/util/data/EmailLinkPersistenceManager.java

```diff
@@ -108,44 +108,44 @@ public void clearAllData(@NonNull Context context) {
 
     /** Holds the necessary information to complete the email link sign in flow */
     public static class SessionRecord {
-        private String sessionId;
```

Comment:
FYI I personally don't get caught up with `mNotation` on something like this which is a pure-Java class with no Android ties.  But LGTM